### ISSUE TITLE
Email confirmation

### ISSUE THREAD

**Issue by clemsos** [topogram/topogram#112]
Need an email confirmation (and an email provider) for user registration

**Comment by almereyda** [comment]
Is this why I cannot login after registration at the Demo?
The deployment script takes a mail uri parameter, so this is probably already solved, but not completely online, yet?

**Comment by clemsos** [comment]
:crying_cat_face:  
sorry, there is no emailing in the current release. 

Do you use the online version ? 
* you can register here : https://app.topogram.io/sign-up
* then sign in here https://app.topogram.io/sign-in

**Comment by almereyda** [comment]
Oh, that's a pity. Didn't it tell me to check my mail for a verification code?
Also visual feedback of successful or denied login could be more clear.
The icon in the bottom right may be different for login and logout, i.e. arrow into the door, and out of it.
But yes, login works and I can create Topograms.

**Comment by clemsos** [comment]
Yes, I should definitely fix the icons, it is totally unclear right now. 
All the text is default from the Meteor users/login package, but needs an update as well.
Thanks for the feedback !
